hudi-client/hudi-client-common/src/test/resources/hive_bucket_id_check.csv

```diff
@@ -0,0 +1,5017 @@
+true,4284,1156332,0.3907772467840961,0.13076037619238812,96
+false,4285,232001,0.7989298469113876,0.1972417493702524,51
+false,4286,326802,0.7197120447108701,0.04042127734961498,97
+true,4287,504134,0.5909979185932502,0.9822547573390213,75
+false,4288,718159,0.26195049105014334,0.5472458604574069,32
+false,4289,1526726,0.17711871897039233,0.575277094138878,99
+true,4290,812750,0.9140973013499697,0.3796649368890421,63
+false,4291,1317050,0.24248004383264465,0.7410156063994852,75
+false,4292,1643125,0.6085366183708281,0.7063448029720921,80
+true,4293,447731,0.12821260164998605,0.203401667219406,47
+false,4294,1714081,0.8793688507458747,0.8794595525908475,4
+false,4295,1643812,0.12313271508220813,0.33501792661998253,62
+true,4296,1383724,0.8183609947896225,0.8429038298773973,74
+false,4297,335896,0.8321438591938564,0.2865772790313237,43
+false,4298,1625373,0.05638362829390897,0.6599540322966345,25
+true,4299,1350139,0.15651758567843754,0.5735758792963424,17
+false,4300,1985367,0.9279377274743309,0.4743725477172783,86
+false,4301,423090,0.952646202028661,0.5873054434744973,83
+true,4302,955344,0.5396785578627711,0.2332481527693966,97
+false,4303,72678,0.9615622673517175,0.7656730026961165,55
+false,4304,1937968,0.3854510139724332,0.08778693547790106,41
+true,4305,1652766,0.22127012777310728,0.32935395779382537,70
+false,4306,1559111,0.8675284262584227,0.099462733463489,37
+false,4307,922429,0.4502982512407431,0.6331031155564117,33
+true,4308,1725312,0.0673286305774844,0.9444454489792443,25
+false,4309,1015075,0.5401191025411652,0.535821354862127,54
+false,4310,129126,0.5106555610357595,0.6252629212104512,72
+true,4311,1504361,0.22717779403672012,0.12495273048661881,21
+false,4312,1172021,0.19959155454980082,0.8149669148791903,43
+false,4313,185086,0.29131536106094025,0.03213221754586304,64
+true,4314,1706421,0.5990250504185136,0.40373249220999796,34
+false,4315,1251543,0.9159264324910478,0.1683939882581419,34
+false,4316,290472,3.1459880495454673E-4,0.716177292112202,3
+true,4317,1037842,0.3494781579625472,0.07553230056434257,44
+false,4318,1952116,0.5796891968949107,0.5241370222031737,89
+false,4319,835036,0.7502858706471437,0.6526664591044923,95
+true,4320,751012,0.1331640077988544,0.20660643818493396,21
+false,4321,1423790,0.7844428535023054,0.3757392803635372,5
+false,4322,1480003,0.32902109282478476,0.4300619006266282,90
+true,4323,1109105,0.09520749829403208,0.5312770178136718,96
+false,4324,1799502,0.09515606264090559,0.9889671434696926,78
+false,4325,241528,0.3907759004175577,0.7207023762402495,7
+true,4326,1920893,0.872021287922155,0.006389083896302439,5
+false,4327,416922,0.8152980170045094,0.41296491896719556,6
+false,4328,733946,0.6260264146655252,0.48485603694057733,0
+true,4329,733532,0.6628716966238256,0.7140616198749685,23
+false,4330,1956003,0.7879164384812248,0.7871551087042735,38
+false,4331,1239587,0.6565996423539073,0.10640163386619472,24
+true,4332,380900,0.860977145579482,0.150321088560726
```

Comment:
any reason we need so much items in this file?

Reply:
select the top 10 records as test case